commit 28be7783bd7fa91b929c5c5631871019263814f4
Author: Michael Zoran <mzoran@crowfest.net>
Date:   Sun Jan 15 07:31:59 2017 -0800

    ARM64: Enable RTL8187/RTL8192CU wifi in build config
    
    These drivers build now, so they can be enabled back
    in the build configuration just like they are for
    32 bit.
    
    Signed-off-by: Michael Zoran <mzoran@crowfest.net>

diff --git a/arch/arm64/configs/bcmrpi3_defconfig b/arch/arm64/configs/bcmrpi3_defconfig
index aae33b4b3c3e..b7d762df19b8 100644
--- a/arch/arm64/configs/bcmrpi3_defconfig
+++ b/arch/arm64/configs/bcmrpi3_defconfig
@@ -528,6 +528,8 @@ CONFIG_RT2800USB_RT3573=y
 CONFIG_RT2800USB_RT53XX=y
 CONFIG_RT2800USB_RT55XX=y
 CONFIG_RT2800USB_UNKNOWN=y
+CONFIG_RTL8187=m
+CONFIG_RTL8192CU=m
 CONFIG_USB_ZD1201=m
 CONFIG_ZD1211RW=m
 CONFIG_MAC80211_HWSIM=m
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1108_28be7783bd7fa91b929c5c5631871019263814f4.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1108_28be7783bd7fa91b929c5c5631871019263814f4.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
